test/malli/json_schema_test.cljc

```diff
@@ -25,11 +26,39 @@
    [[:map
      [:a string?]
      [:b {:optional true} string?]
-     [:c {:optional false} string?]] {:type "object"
-                                      :properties {:a {:type "string"}
-                                                   :b {:type "string"}
-                                                   :c {:type "string"}}
-                                      :required [:a :c]}]
+     [:c {:optional false} string?]]
+    {:type "object"
+     :properties {:a {:type "string"}
+                  :b {:type "string"}
+                  :c {:type "string"}}
+     :required [:a :c]}]
+   [[:map
+     [:x :int]
+     [::m/default [:map-of :int :int]]]
+    {:type "object"
+     :properties {:x {:type "integer"}}
+     :required [:x]
+     :additionalProperties {:type "integer"}}]
+   [[:map
+     [:x :int]
+     [::m/default [:fn {:json-schema/default {:x 1}, :gen/gen (gen/return {})} map?]]]
```

Comment:
how can the `::m/default` be an `:fn` schema? what does that mean?

Reply:
you can say "... and the map should have keys :x and :y and they should be the same".